### PR TITLE
more uwgl for norway

### PR DESCRIPTION
**Pull request by FredrikAreschoug**
To any reviewer this is my an Tomas conversation:
```Tomas  1:55 PM
Det är samma uwgl som på lägenhet i Sverige, dvs inte fler än 6 pers per hushåll eller lägenhet större än 250 kvm.
På youth är det max två pers på försäkringen och inte större lägenhet än 50kvm.
1:55
Reise för grownups är 6 pers, på youth max en om man ska få youthpriset

Fredrik Areschoug  1:57 PM
okej grymt där va det ju några grejer vi hade missat :blush: och ungdom är under 35?

Tomas  1:58 PM
Tror vi landade på 30 men har inte varit med i dom diskussionerna. Dubbelkolla med Kajsa :blush:

Fredrik Areschoug  1:58 PM
will do :slightly_smiling_face:

Fredrik Areschoug  2:09 PM
bara för att var helt säker om man ska få ungdoms priset på reseförsäkringen ska man teckna den själv?

Tomas  2:09 PM
Yes det stämmer. Vi fick inte ha fler än en på reseförsäkringen och fastpris (youth)

Fredrik Areschoug  2:10 PM
cool, tack :slightly_smiling_face:
```
Did I miss anything?

### CODE REVIEW

File: src/main/kotlin/com/hedvig/underwriter/service/guidelines/NorwegianHomeContentsGuidelines.kt [hunk 1]
```diff
@@ -10,45 +10,68 @@ object NorwegianHomeContentsGuidelines {
         NorwegianHomeContentLivingSpaceAtLeast1Sqm,
         NorwegianHomeContentscoInsuredNotMoreThan5,
         NorwegianHomeContentsLivingSpaceNotMoreThan250Sqm,
-        NorwegianYouthHomeContentsAgeNotMoreThan34Years
+        NorwegianYouthHomeContentsLivingSpaceNotMoreThan50Sqm,
+        NorwegianYouthHomeContentsAgeNotMoreThan30Years,
```
Comment (meletios) [inline src/main/kotlin/com/hedvig/underwriter/service/guidelines/NorwegianHomeContentsGuidelines.kt:14]: Is this applicable to Youth?

Comment (meletios) [reply src/main/kotlin/com/hedvig/underwriter/service/guidelines/NorwegianHomeContentsGuidelines.kt:14]: only

Comment (meletios) [reply src/main/kotlin/com/hedvig/underwriter/service/guidelines/NorwegianHomeContentsGuidelines.kt:14]: Should we have a separate  `NorwegianYouthHomeContentsGuidelines`

Comment (FredrikAreschoug) [reply src/main/kotlin/com/hedvig/underwriter/service/guidelines/NorwegianHomeContentsGuidelines.kt:14]: yes?

**Pull request merged**